coda-automation/scripts/delegation-helper.py

```diff
@@ -0,0 +1,30 @@
+import jsonlines 
```

Comment:
There are some interesting scripts throughout. Worth getting together to discuss them and see what we can generalize + re-use?